src/main/scala/scraml/libs/TapirPackageDefinitions.scala

```diff
@@ -0,0 +1,77 @@
+package scraml.libs
+
+import scala.meta._
+
+import scraml.{MetaUtil, ModelGenContext}
+
+final class TapirPackageDefinitions(private val context: ModelGenContext) {
+  // Scala v2.12 specific tapir definitions
+  private object Scala212 {
```

Comment:
Do you have an project that wants to use `scraml` with 2.12 or did this come from the default Scala version in the sbt tests?
I am not to eager to support 2.12 and would rather work towards 3 now, so it is nice to see how you solved cross version generation. Feel free to keep it as is.